### PR TITLE
chore: updates fhevmjs and fhevmjsMocked to v0.6

### PR DESCRIPTION
**Pull request by jatZama**

### CODE REVIEW

File: test/instance.ts [hunk 1]
```diff
@@ -13,13 +21,18 @@ const FHE_CLIENT_KEY_PATH = process.env.FHE_CLIENT_KEY_PATH;
 
 let clientKey: Uint8Array | undefined;
 
+const kmsAdd = dotenv.parse(fs.readFileSync('lib/.env.kmsverifier')).KMS_VERIFIER_CONTRACT_ADDRESS;
```
Comment (immortal-tofu) [inline test/instance.ts:24]: Maybe using dotenv package would be cleaner https://www.npmjs.com/package/dotenv

Comment (immortal-tofu) [inline test/instance.ts:24]: Maybe use 

```
const kmsAdd = dotenv.config({ path: 'lib/.env.kmsverifier' }).KMS_VERIFIER_CONTRACT_ADDRESS;
const aclAdd = dotenv.config({ path: 'lib/.env.acl' }).ACL_CONTRACT_ADDRESS;
```

File: test/instance.ts [hunk 2]
```diff
@@ -13,13 +21,18 @@ const FHE_CLIENT_KEY_PATH = process.env.FHE_CLIENT_KEY_PATH;
...
-  });
-  instance.reencrypt = reencryptRequestMocked;
-  instance.createEncryptedInput = createEncryptedInputMocked;
-  instance.getPublicKey = () => '0xFFAA44433';
+  const instance = {
+    reencrypt: reencryptRequestMocked,
+    createEncryptedInput: createEncryptedInputMocked,
+    getPublicKey: () => '0xFFAA44433',
+    generateKeypair: generateKeypair,
+    createEIP712: (publicKey: string, verifyingContract: string, delegatedAccount?: string) =>
```
Comment (immortal-tofu) [inline test/instance.ts]: Maybe rewrite as follow
```
createEIP712: createEIP712(network.config.chainId),
```

**Review by immortal-tofu — CHANGES_REQUESTED** [review]

**Review by immortal-tofu — APPROVED** [review]

File: test/instance.ts (hunk 1 above)
Comment (jatZama) [reply test/instance.ts:24]: i tried but was unsuccessful, will do this later in Q4.

File: test/instance.ts (hunk 2 above)
Comment (jatZama) [reply test/instance.ts]: done

**Pull request merged**